### PR TITLE
Increase buffer for playback after rebuffer

### LINKED ISSUE #6425: Buffering while playing videos
### Checklist

- [x] I am using the latest version - 0.21.3 
- [x ] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x ] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x ] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Actual behaviour

The problem of buffering while playing videos using wifi ..other apps like Youtube works fine..it sometimes occur and other times it is fine.. the problem both occurs in full view mode and pip mode. To an extent, it buffers, plays for a few seconds and buffers again.

### Expected behavior

I expect the app to be smooth  as using wifi does not hinder the YouTube app. Besides the buffering im loving this app..better features and lightweight than You tube...keep up the good work!

### Device info

 - Android version/Custom ROM version: stock android 9(pie)
 - Device model: Asus ZenFone Max Pro M2

### PR DESCRIPTION
**Pull request by Redirion**
#### What is it?
- [X] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

NewPipe used a very small/short buffer of just 500ms to fill after a buffer depletion. Some devices seem to have issues with that and get in a loop of draining this buffer and having to refill it again leading to a short buffering every 500ms.
From ExoPlayer doc:
`The default duration of media that must be buffered for playback to resume after a rebuffer, in milliseconds. A rebuffer is defined to be caused by buffer depletion rather than a user action`
We are using ExoPlayer's default value now which is 5000ms.

#### Fixes the following issue(s)

- Fixes #6425 

#### APK testing 

On the website the APK can be found by going to the "Checks" tab below the title and then on "artifacts" on the right.

#### Due diligence
- [X] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by Stypox — APPROVED** [review]

**Review by TobiGr — APPROVED** [review]
Looks good.

Let's wait for feedback from the people in the original issue

**Comment by opusforlife2** [comment]
Make it an even 10000ms, I say! “ψ(｀∇´)ψ

**Comment by bibin-v2020** [comment]
So far so good... Just started testing with new apk

**Comment by Flam3z** [comment]
Just downloaded the apk and testing out a handful of videos. Issue seems to be fixed :)

**Comment by Slicer32** [comment]
Sorry for the late reply, i have tried thw newpipe increasebuffer after apk and thankfully it works wonders...very smooth indeed!

**Comment by bibin-v2020** [comment]
Yes!! fine now. I can see considerable difference on buffering! It's smooth like never before.. Thanks

**Comment by bibin-v2020** [comment]
Will the fix be added with v21.4? I can see it's already released but I didn't see about 'buffer' in the latest blog post.

**Comment by SmoothJaz** [comment]
> @Slicer32 @nzimas @SmoothJaz @XNUConner @castrik @bibin-v2020 @Flam3z @snsgnq could you test if [this APK](https://github.com/TeamNewPipe/NewPipe/suites/2927617366/artifacts/65718983) (from [the CI](https://github.com/TeamNewPipe/NewPipe/pull/6444/checks?check_run_id=2761566133)) fixes your buffering issues? (I pinged you since you commented in #6425)

Bit late to this party but I haven't had any issue so far with this apk!

**Comment by Stypox** [comment]
@bibin-v2020 rhe latest blog post was released before this was merged

**Comment by bibin-v2020** [comment]
> @bibin-v2020 rhe latest blog post was released before this was mergOk.. Anyway, thanks!! Much Appreciated for your effort.

> @bibin-v2020 rhe latest blog post was released before this was merged

👍

**Comment by opusforlife2** [comment]
@SmoothJaz A little heads-up: when you quote a text block that contains mentions of other users in it, those people get notifications all over again. Try to avoid that.

**Comment by SmoothJaz** [comment]
@opusforlife2 oops, how embarrassing!

**Comment by JoostBaars** [comment]
I just installed V0.21.5 yesterday and hoped the buffering issue would've been resolved but I just noticed it again. It is still about half a second until it buffers again. For me it doesn't always happen, often force-closing the app or deleting the cache helps for some hours.

**Comment by mehedihasanziku** [comment]
seems like this fix an issue with download, like if we try to download multiple video by navigating next and next, some times it stuck with download option and show "Force Close" or "Wait", but now it seems working smoothly, I will update if any issues related to this.

Thanks for your affort

**Comment by Scrayil** [comment]
Hi, in the last month i experienced this issue several times.
I've just cleared the cache of the application and it seems this has actually solved the problem for now.

**Comment by opusforlife2** [comment]
@Scrayil This is not the right place to add your comments. They would be better suited to #6510.

**Pull request merged**
